experimenter/experimenter/experiments/constants.py

```diff
@@ -124,6 +125,19 @@ class ApplicationConfig:
     randomization_unit=BucketRandomizationUnit.NIMBUS,
 )
 
+APPLICATION_CONFIG_MONITOR_WEB = ApplicationConfig(
```

Comment:
This is going to be our first application that isnt available in multiple versions. Right now the version field really only applies to desktop and mobile browsers.

We should add a flag to ApplicationConfig that indicates whether or not it is versioned. We will also want to omit the min_firefox_version and max_firefox_version fields on the Audience page.

The v5 serializer should be updated to make sure that you cannot set the min_firefox_version or max_firefox_version for unversioned apps.

cc @jaredlockhart does this all sound correct?

Reply:
yes that all will be done in a separate issue- making sure all the fields applicable to monitor renders that only